sdk/textanalytics/azure-ai-textanalytics/src/main/java/com/azure/ai/textanalytics/implementation/Utility.java

```diff
@@ -314,7 +334,7 @@ public static String parseOperationId(String operationLocation) {
         if (!CoreUtils.isNullOrEmpty(operationLocation)) {
             int lastIndex = operationLocation.lastIndexOf('/');
             if (lastIndex != -1) {
-                return operationLocation.substring(lastIndex + 1);
+                return operationLocation.substring(lastIndex + 1, lastIndex + 37);
```

Comment:
We should generally avoid the use of [magic number ](https://stackoverflow.com/questions/47882/what-is-a-magic-number-and-why-is-it-bad) and replace it with what it stands for.